apis/cluster/v1/membercluster_types.go

```diff
@@ -17,6 +17,9 @@ import (
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="Joined")].status`,name="Joined",type=string
 // +kubebuilder:printcolumn:JSONPath=`.metadata.creationTimestamp`,name="Age",type=date
+// +kubebuilder:printcolumn:JSONPath=`.status.properties.kubernetes-fleet\.io/node-count.value`,name="NodeCount",type=string
+// +kubebuilder:printcolumn:JSONPath=`.status.resourceUsage.allocatable.cpu`,name="AllocatableCPU",type=string
```

Comment:
I think we need available instead of allocatable

Reply:
or both

Reply:
Allocatable will be part of -o wide and available part of regular.